metadata_service/proxy/neo4j_proxy.py

```diff
@@ -518,7 +518,7 @@ def add_owner(self, *,
 
         upsert_owner_relation_query = textwrap.dedent("""
         MATCH (n1:User {key: $user_email}), (n2:Table {key: $tbl_key})
-        MERGE (n2)-[r2:OWNER]->(n1)
+        MERGE (n1)-[r1:OWNER_OF]->(n2)-[r2:OWNER]->(n1)
```

Comment:
i don't think you need to have bidirection in this query?

Reply:
I think we would need bidirectional relation here as we are adding new relation between a user and table. Else we will again create duplicate relationships.
e.g From UI we created `Table-OWNER->User` relationship.  Now from backend we again create relationship between same table and user as 'Table-OWNER->User-OWNER_OF->Table', at this time neo4j graph will not find whole bidirectional relation hence it will create a pair of `OWNER and OWNER_OF ` relation, which will result into two `OWNER` relation and one `OWNER_OF` relationship between same user and table.
This is one case I could think of, but I feel there may be other cases as well because `MERGE` in neo4j graph looks for complete match.